packages/core/src/plugins/stream/SoundcloudPlugin.ts

```diff
@@ -12,13 +12,14 @@ class SoundcloudPlugin extends StreamProviderPlugin {
     this.sourceName = 'Soundcloud';
     this.description = 'Allows Nuclear to find music streams on Soundcloud';
     this.image = null;
+    this.isDefault = true;
   }
 
   resultToStream(result): StreamData {
     return {
-      source: this.sourceName,
+      source: 'Soundcloud',
```

Comment:
for some reason this property will not pick up sourceName from its instance. Hard coded for now.

Reply:
Looks like `this` won't work here because it's not bound (this isn't an arrow function). If you change it to an arrow function, you should be able to revert this.